Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build stage
+FROM node:16-alpine as builder
+WORKDIR /etc/logto
+ENV CI=true
+COPY . .
```

Comment:
We can add `package.json` and `pnpm-lock.yaml` first, then it'll have 3 layers:
1. add package and lock file
2. pnpm i
3. add all other files

Then we can take advantage of docker layer cache.

But is not a blocker.

Reply:
Yes, this is preferred approach. 👍

Reply:
LOG-3197

Reply:
kind of tricky in monorepo, do it later